src/components/shared/HomeSlider.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import { RxDotFilled } from 'react-icons/rx';
+import { SlidingImages } from '../../utils/SlidingImages';
+
+type Images = {
+  images: SlidingImages;
+};
+
+export function HomeSlider({ images }: Images) {
+  const [imageIndex, setIndex] = useState(0);
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      if (imageIndex < images.length - 1) {
+        setIndex(imageIndex + 1);
+      } else {
+        setIndex(0);
+      }
+    }, 3000);
+
+    return () => clearInterval(intervalId);
+  }, [imageIndex, images.length - 1]);
+
+  const goToSlide = (slideIndex: number) => {
+    setIndex(slideIndex);
+  };
+
+  return (
+    <div className="sm:w-full w-full h-[90vh] m-auto  relative">
+      <div
+        style={{ backgroundImage: `url(${images[imageIndex]})` }}
+        className="h-full bg-center bg-cover duration-500 "
+      />
+      <div className="absolute top-[40vh] right-0 left-0 flex height-full justify-center items-center">
+        <div className="flex flex-col items-center ">
+          <h1 className="text-white font-semibold text-5xl py-2">
+            Welcome to Team Cypher
+          </h1>
+          <h1 className="text-white font-medium text-3xl">
+            {' '}
+            official e-commerce website
+          </h1>
+        </div>
+      </div>
+      <div className="absolute bottom-3 right-0 left-0 flex justify-center items-end">
+        {images.map((_, slideIndex) => (
+          <button type="button" key={0} onClick={() => goToSlide(slideIndex)}>
```

Comment:
- I like the fact that the images slide automatically. Something else though, if you check on figma there left and right buttons on the slider, i think there are more second nature than clicking on those icons, but that functionality of clicking on the icons should also remain